File: Client/cinema-app/src/app/core/models/cinema.ts
import { CinemaFavor } from '@core/models/cinema-favor';
import { Hall } from '@core/models/hall';

export interface Cinema {
  id?: number,
  name: string,
  address: string,
  cityId: number,
  halls: Hall[],
  cinemaFavors: CinemaFavor[]
}

File: Server/Cinema/Cinema.Domain/Entities/Cinema.cs
namespace CinemaApplication.Domain.Entities
{
    public class Cinema
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public short CityId { get; set; }
        public City City { get; set; }
        public ICollection<Hall> Halls { get; set; }
        public ICollection<CinemaFavors> CinemaFavors { get; set; }
    }
}